lib/creek/styles/converter.rb

```diff
@@ -45,16 +46,18 @@ def self.call(value, type, style, options = {})
         when 'b'
           value.to_i == 1
         when 'str'
-          value
+          convert_html_entities(value)
```

Comment:
Try to create a xlsx file where a cell contain a ampersand e look what happen to the value returned when parsing the xml. You will have a &amp; instead of the converted one &, as represented in the excel. 
This is surely a rude way to fix the problem (I'm releasing a software that use your gem in the short) but the thing persist. Am I using the gem wrongly, maybe? There is something I have missed in my usage, maybe? Thanks